lib/const.rb

```diff
@@ -2,7 +2,7 @@
 # Defines common constants used in different parts of crew
 require 'etc'
 
-CREW_VERSION = '1.45.0'
+CREW_VERSION = '1.45.1'
```

Comment:
This should be 1.49.9 since the last version was 1.49.8.

Reply:
i don't understand what you mean-- the current version is 1.50.0 in this PR, and the previous PR went from 1.49.9 to 1.50.0

Reply:
This PR downgraded crew from 1.49.9 to 1.45.0.

Reply:
ohhhh i can't read and i did the version number wrong oops